src/git-sync

```diff
@@ -30,15 +30,20 @@ function preconditions {
 
 
 function skip_message_prefix {
-  echo "To skip the sync of the '$(get_current_branch_name)' branch"
+  local branch_name=$(get_current_branch_name)
+  if [ "$(rebase_in_progress)" = true ]; then
```

Comment:
If you are rebasing, the branch name is HEAD. Thus if there are conflicts while rebasing I need to do something special to determine the branch name.